dbt/include/trino/macros/catalog.sql

```diff
@@ -1,63 +1,127 @@
+            {{ trino__get_catalog_schemas_where_clause_sql(schemas) }}
+        ),
+        table_comment as (
+            {{ trino__get_catalog_table_comment_schemas_sql(information_schema, schemas) }}
+        )
+        {{ trino__get_catalog_results_sql() }}
+    {%- endset -%}
+
+    {{ return(run_query(query)) }}
 
-            select
-                table_catalog as "table_database",
-                table_schema as "table_schema",
-                table_name as "table_name",
-                table_type as "table_type",
-                null as "table_owner"
+{%- endmacro %}
 
-            from {{ information_schema }}.tables
-            where
-                table_schema != 'information_schema'
-                and
-                table_schema in ('{{ schemas | join("','") | lower }}')
 
-        ),
+{% macro trino__get_catalog_relations(information_schema, relations) -%}
 
+    {% set query %}
+        with tables as (
+            {{ trino__get_catalog_tables_sql(information_schema) }}
+            {{ trino__get_catalog_relations_where_clause_sql(relations) }}
+        ),
         columns as (
+            {{ trino__get_catalog_columns_sql(information_schema) }}
+            {{ trino__get_catalog_relations_where_clause_sql(relations) }}
+        ),
+        table_comment as (
+            {{ trino__get_catalog_table_comment_relations_sql(information_schema, relations) }}
+        )
+        {{ trino__get_catalog_results_sql() }}
+    {%- endset -%}
 
-            select
-                table_catalog as "table_database",
-                table_schema as "table_schema",
-                table_name as "table_name",
+    {{ return(run_query(query)) }}
 
-                column_name as "column_name",
-                ordinal_position as "column_index",
-                data_type as "column_type",
-                comment as "column_comment"
+{%- endmacro %}
 
-            from {{ information_schema }}.columns
-            where
-                table_schema != 'information_schema'
-                and
-                table_schema in ('{{ schemas | join("','") | lower }}')
 
-        ),
+{% macro trino__get_catalog_tables_sql(information_schema) -%}
+    select
+        table_catalog as "table_database",
+        table_schema as "table_schema",
+        table_name as "table_name",
+        table_type as "table_type",
+        null as "table_owner"
+    from {{ information_schema }}.tables
+{%- endmacro %}
+
+
+{% macro trino__get_catalog_columns_sql(information_schema) -%}
+    select
+        table_catalog as "table_database",
+        table_schema as "table_schema",
+        table_name as "table_name",
+        column_name as "column_name",
+        ordinal_position as "column_index",
+        data_type as "column_type",
+        comment as "column_comment"
+    from {{ information_schema }}.columns
+{%- endmacro %}
+
+
+{% macro trino__get_catalog_table_comment_schemas_sql(information_schema, schemas) -%}
```

Comment:
adjust to https://github.com/starburstdata/dbt-trino/pull/357/

Reply:
As discussed with findepi, we should wait for the moment. I wouldn't rush to include it in the 1.7.0 release. Perhaps we could make adjustments in the next patch release? WDYT?

Reply:
yup, discussed offline